coalib/results/AbsolutePosition.py

```diff
@@ -0,0 +1,55 @@
+from coalib.results.TextPosition import TextPosition
+from coalib.misc.Decorators import enforce_signature
+
+
+class AbsolutePosition(TextPosition):
+
+    @enforce_signature
+    def __init__(self,
+                 text: (tuple, list, None)=None,
+                 position: (int, None)=None):
+        """
+        Creates an AbsolutePosition object that represents the index of a
+        character in a string.
+
+        :param text:     The text containing the character.
+        :param position: Position identifying the index of character
+                         in text.
+        """
+        line = column = None
+        if position and text:
+            line, column = calc_line_col(text, position)
+        self._text = text
+        self._position = position
+        super().__init__(line, column)
+
+    @property
+    def position(self):
+        return self._position
+
+
+def calc_line_col(text, position):
```

Comment:
Oh, I see. It is solely used in the class, so it should probably,be it's "private" method. Why present it to the global name space? Even more it's so small and has only one invocation, that maybe it'll be better to just put this code into `__init__`?